src/table/boot.rs

```diff
@@ -25,7 +25,7 @@ pub struct BootServices {
         addr: &mut u64,
     ) -> Status,
     free_pages: extern "win64" fn(addr: u64, pages: usize) -> Status,
-    memory_map: extern "win64" fn(
+    get_memory_map: unsafe extern "win64" fn(
```

Comment:
Probably best to keep the spec name for this entry point, even if we call it differently on our side.